atom/common/platform_util_mac.mm

```diff
@@ -252,4 +248,91 @@ void Beep() {
   NSBeep();
 }
 
+#if defined MAS_BUILD
+void SetProcessTitleActivityMonitor(const std::string& name) {
+  // NB: Can't be implemented with public APIs
+}
+#else
+void SetProcessTitleActivityMonitor(const std::string& name) {
+  if (name.length() < 1) {
+    NOTREACHED() << "SetProcessTitleActivityMonitor given bad name.";
```

Comment:
`NOTREACHED` means to crash, should not be used in a path route that may be executed.